app/components/news/Block.vue

```diff
@@ -46,13 +58,24 @@ export default {
   },
   mounted () {
     this.loadNewsImages(this.news.featured_media)
+    if (this.news.tags.length) {
+      this.loadNewsTags(this.news.tags)
+    }
   },
   methods: {
     async loadNewsImages (newsId) {
       const adress = `${contentApiURL}/media/${newsId}`
       const responseNews = await this.$axios.get(adress)
       this.newsImage = responseNews.data?.media_details?.sizes?.large?.source_url || responseNews.data?.source_url || this.newsImage
       this.newsImageSmall = responseNews.data?.media_details?.sizes?.medium_large?.source_url || responseNews.data?.source_url || this.newsImage
+    },
+    async loadNewsTags (tagIDs) {
+      const adress = `${tagsURL}?include=${tagIDs}`
```

Comment:
Small typo here, it's address

Reply:
ok, thanks, I'll merge it for now, so you don't have to reapprove, but I'll change it in some revision